src/core/search/searchapi.js

```diff
@@ -242,4 +242,36 @@ export default class SearchApi {
     return request.get()
       .then(response => response.json());
   }
+
+  /**
+ * Initiate a search
+ * @param {boolean} includeLastName include last name in the search
+ * @param {boolean} useUserName use username instead of real name
+ * @param {string} additionalSearchText more text to add to the search
+ */
+Search(includeLastName, useUserName, additionalSearchText) {
```

Comment:
Based on what I recall from the codelab, I think the function name is supposed to have your name in it. Something like 'chrisSearch'. For function names, we use camelCase so we don't capitalize the first letter

Reply:
Wow, good call Connor. I'll be sure to make the changes.

Reply:
The fixed have been implemented. Thoughts?

Reply:
looks good!